From 7619530bf92fac7dc649e06388fd02508d2766bc Mon Sep 17 00:00:00 2001
From: Matt Madison <matt@madison.systems>
Date: Sat, 7 Nov 2020 03:31:11 -0800
Subject: [PATCH] board/nvidia/p3541-0000: fix use of bd_t

Convert the use of bd_t to struct bd_info in
the board initialization file.

Signed-off-by: Matt Madison <matt@madison.systems>
---
 board/nvidia/p3541-0000/p3541-0000.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/board/nvidia/p3541-0000/p3541-0000.c b/board/nvidia/p3541-0000/p3541-0000.c
index 52a283fc7e..8080103386 100644
--- a/board/nvidia/p3541-0000/p3541-0000.c
+++ b/board/nvidia/p3541-0000/p3541-0000.c
@@ -170,7 +170,7 @@ static void ft_carveout_setup(void *fdt)
 	}
 }
 
-int ft_board_setup(void *fdt, bd_t *bd)
+int ft_board_setup(void *fdt, struct bd_info *bd)
 {
 	ft_mac_address_setup(fdt);
 	ft_carveout_setup(fdt);
-- 
2.25.1

